lib/galaxy/config.py

```diff
@@ -279,6 +279,7 @@ def __init__( self, **kwargs ):
         self.allow_library_path_paste = kwargs.get( 'allow_library_path_paste', False )
         self.disable_library_comptypes = kwargs.get( 'disable_library_comptypes', '' ).lower().split( ',' )
         self.watch_tools = kwargs.get( 'watch_tools', 'false' )
+        self.lookup_tools_in_other_tool_sheds = kwargs.get( 'lookup_tools_in_other_tool_sheds', 'false' )
```

Comment:
OK, noted, but I'll do as @jmchilton said and remove the configurability, if there are no further objections.